src/mcpHelpers.js

```diff
@@ -18,7 +18,15 @@ const mcpCustomizr = new CustomizrClient({
  * @return {Promise<*|void>}
  */
 async function getMcpSettings(accessToken) {
-    return await mcpCustomizr.getSettings(accessToken);
+    let data = await mcpCustomizr.getSettings(accessToken);
+    if (Object.keys(data).length === 0) {
+        return {
+            language: ['en'],
+            regionalSettings: 'en',
+            timezone: 'America/New_York',
```

Comment:
Does it mean that when I use this client, I have no chance to detect if these are "default" or settings set directly by the user?

What If I want to default to browser locale if Customizr client does not return any settings? I cannot do that anymore, because Customizr always returns something.

Reply:
What do you do in case of "default" in your UI? The goal for this package is to help maintain consistent behaviour across all of the applications and not have to copy paste code around. Do you have different defaults between application @akincel ?

Reply:
Can we read something from the browser, like the browser language or time zone the user is in?

Reply:
@akincel If you want to use the "default" and differentiate 'no info' vs 'default', you are still free to do so and the package still provides this option. Instead of using the "mcp helper", you can just use the client and get the data directly. Would that work for you in an acceptable way? I'm happy to improve the code of course ;)

The purpose of this helper method is to do as much as possible so the client doesn't have to. Before, returning 'empty' was causing issues as most UI were expecting data and not really checking all possible data for correctness or existence (possibly wrong assumption, I agree). 

Additionally, since we don't really have new users every day, these type of errors were hard to spot/reproduce (to some extent). Yes, we can create a new user but once this user sets the data, we cannot reproduce this any more and we'd need to create another user. Customizr is not providing a way to remove the data so the 404 case is only happening once per user (as far as I know).

@thoean We can try. It seems that browsers do have different way to represent the language though, so we'd have to additionally convert it to the format that Customizr expects (2-letter). Also it seems (based on very quick search that might be wrong) that sometimes browsers might report their language instead of the user's preferred one, so the value of this might decrease a bit. Same is valid for the other two properties - regional settings and timezone. 

Anyway, I'd be happy to find a better solution and possibly the right place for it - it could be this package, it could be the shared react settings package (which uses cimpress-customizr) or it might be within each app that cares about this ;)

Reply:
Thank you for explaining your intention. I agree with the consistency across applications. For new users it is about these two options:
- a) a user goes to a UI and sees everything in English, US time formats and time-shifted; the user needs to figure out where to change these settings
- b) a user goes to a UI and ideally sees everything in the local language with understandable time format; the user even does not need to know about settings

Anyway, I think what you did is a good first step. I checked our applications. We are providing the same defaults as you did here. Getting the settings from browser sounds interesting. When taking into consideration that doing that is not so straightforward, do you see it as something worth investing into?